pages/flow.vue

```diff
@@ -1,56 +1,457 @@
+              @click="onClickAnchor"
+            >
+              <span>{{ $t('軽い症状があり不安のある方') }}</span>
+              <fig-cond-anx :class="$style.fig" aria-hidden="true" />
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div id="sydr" :class="$style.section">
+        <h4 :class="$style.heading">
+          {{ $t('かかりつけ医がいて、次の症状がある方') }}
+        </h4>
+        <ul :class="$style.boxes">
+          <li :class="[$style.box, $style.border]">
+            <span :class="$style.lead">{{
+              $t('風邪のような症状や、37.5℃以上の発熱が続いている')
+            }}</span>
+            <span :class="$style.alignLeft">{{
+              $t(
+                'ご高齢の方・基礎疾患のある方・妊娠中の方は2日程度続いた場合、それ以外の方は4日以上続いた場合を相談の目安としてください。'
+              )
+            }}</span>
+          </li>
+          <li :class="[$style.box, $style.border]">
+            <span :class="$style.lead">{{
+              $t('強いだるさ（倦怠感）や息苦しさ（呼吸困難）がある')
+            }}</span>
+          </li>
+        </ul>
+        <p :class="[$style.box, $style.bgYellow]">
+          <span :class="$style.large">{{
+            $t('日ごろ受診されている医療機関に電話でご相談ください。')
+          }}</span>
+          <span :class="$style.small">{{
+            $t(
+              '（上記の症状がない場合も、軽い症状があるなど不安のある方は、日ごろ受診されている医療機関に電話でご相談ください。）'
+            )
+          }}</span>
+        </p>
       </div>
-      <div class="only-pc" aria-hidden="true">
-        <flow-pc />
+      <div id="sy" :class="$style.section">
+        <h4 :class="$style.heading">
+          {{ $t('かかりつけ医がいない、次の症状がある方') }}
+        </h4>
+        <ul :class="$style.boxes">
+          <li :class="[$style.box, $style.border]">
+            <span :class="$style.lead">{{
+              $t('風邪のような症状や、37.5℃以上の発熱が続いている')
+            }}</span>
+            <span :class="$style.alignLeft">{{
+              $t(
+                'ご高齢の方・基礎疾患のある方・妊娠中の方は2日程度続いた場合、それ以外の方は4日以上続いた場合を相談の目安としてください。'
+              )
+            }}</span>
+          </li>
+          <li :class="[$style.box, $style.border]">
+            <span :class="$style.lead">{{
+              $t('強いだるさ（倦怠感）や息苦しさ（呼吸困難）がある')
+            }}</span>
+          </li>
+        </ul>
+        <div :class="[$style.box, $style.bgGray]">
+          <h5 :class="$style.heading">
+            {{ $t('新型コロナ受診相談窓口は、24時間対応しています') }}
+          </h5>
+          <dl :class="$style.contact">
+            <div>
+              <dt>平日（日中）：</dt>
+              <dd>
+                <a
+                  href="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/coronasodan.html"
+                >
+                  {{ $t('各保健所の電話番号は福祉保健局HP') }}
+                  <v-icon
+                    :class="$style.iconExternal"
+                    :aria-label="this.$t('別タブで開く')"
+                    role="img"
+                    :aria-hidden="false"
+                  >
+                    mdi-open-in-new
+                  </v-icon> </a
+                >{{ $t('に掲載しています') }}
+              </dd>
```

Comment:
- `<i18n>` コンポーネントを使用
- 外部リンク表示に `<external-link>` コンポーネントを使用
    - `:icon-size="16"` の数値は変える必要があるかも？
```suggestion
              <dd>
                <i18n path="{publicHealthCenter}に掲載しています">
                  <template v-slot:publicHealthCenter>
                    <external-link
                      url="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/coronasodan.html"
                      :icon-size="16"
                    >
                      {{ $t('各保健所の電話番号は福祉保健局HP') }}
                    </external-link>
                  </template>
                </i18n>
              </dd>
```